tests/test_store/test_core.py

```diff
@@ -4,12 +4,50 @@
 import pytest
 from _pytest.compat import LEGACY_PATH
 
-from zarr.core.common import AccessModeLiteral
+from zarr import Group
+from zarr.core.common import AccessModeLiteral, ZarrFormat
 from zarr.storage import FsspecStore, LocalStore, MemoryStore, StoreLike, StorePath
-from zarr.storage._common import make_store_path
+from zarr.storage._common import contains_array, contains_group, make_store_path
 from zarr.storage._utils import normalize_path
 
 
+@pytest.mark.parametrize("write_group", [True, False])
+@pytest.mark.parametrize("zarr_format", [2, 3])
+async def test_contains_group(local_store, write_group: bool, zarr_format: ZarrFormat) -> None:
+    """
+    Test contains group method
```

Comment:
can we parametrize this over `path`, ensuring that we check a level of nesting? e.g. `@pytest.mark.parametrize('path', ['foo', 'foo/bar'])`

Reply:
and similarly for the `contains_array` tests